.github/actions/verify-submodule-pull-request/Check-Parent.ps1

```diff
@@ -3,12 +3,17 @@
     [string] $Branch
 )
 
-$url = "https://api.github.com/repos/$Repository/pulls?state=open&per_page=100"
-$titles = curl -s -H 'Accept: application/vnd.github.v3+json' $url | ConvertFrom-Json | ForEach-Object { $PSItem.title }
+$ProgressPreference = "SilentlyContinue"
```

Comment:
This isn't used.

Reply:
It's a global override for the current session not to display a progress bar (while the request is executing). It's recommended for automated sessions, but I just checked, and it doesn't look like we use it anywhere else, so GHA doesn't care - I'll remove it.